### ISSUE TITLE
[Bug(?)] Character randomly "faints" or waits infinitely

### ISSUE THREAD

**Issue by harald921** [CleverRaven/Cataclysm-DDA#15616]
So I downloaded the latest experimental about 15 minutes ago, created a new world and all that.

First time, I was in the middle of the forest, foraging. Had not yet encountered a single enemy. Then suddenly... game freeze. Weird. Was about to alt+f4 when the game finally started running again. Dead. Apparently a brute had been mauling me for the past 40 minutes and continued to do so. I assumed that this was a new feature of the brute, that he could stun you and I got repeatedly stunned or something.

Second time, (new char) I was simply going into my shelter when the game froze again, however not for as long as last time. When the game started running again my characters corpse was being mauled by a tough zombie.

TL;DR - character randomly gets infinitely stunned for no (apparent) reason

Edit: Happened now again, about 50 turns since spawn

Edit: And once again. Sigh. I'll check in later to see if there have been any comments

**Comment by Zireael07** [comment]
Duplicate of #15615.
